Shaders/DisplayDepth.fx

```diff
@@ -8,251 +8,426 @@
-			"If the Depth map is shown upside down set it to 1."
-		#define iUIUpsideDown 0
-	#endif
-	
-	#if RESHADE_DEPTH_INPUT_IS_REVERSED
-		#define REVERSED_HELP_TEXT "RESHADE_DEPTH_INPUT_IS_REVERSED is currently set to 1.\n"\
-			"If close objects in the Depth map are bright and far ones are dark set it to 0.\n"\
-			"Also try this if you can see the normals, but the depth view is all black."
-		#define iUIReversed 1
-	#else
-		#define REVERSED_HELP_TEXT "RESHADE_DEPTH_INPUT_IS_REVERSED is currently set to 0.\n"\
-			"If close objects in the Depth map are bright and far ones are dark set it to 1.\n"\
-			"Also try this if you can see the normals, but the depth view is all black."
-		#define iUIReversed 0
-	#endif
-	
-	#if RESHADE_DEPTH_INPUT_IS_LOGARITHMIC
-		#define LOGARITHMIC_HELP_TEXT "RESHADE_DEPTH_INPUT_IS_LOGARITHMIC is currently set to 1.\n"\
-			"If the Normal map has banding artifacts (extra stripes) set it to 0."
-		#define iUILogarithmic 1
-	#else
-		#define LOGARITHMIC_HELP_TEXT "RESHADE_DEPTH_INPUT_IS_LOGARITHMIC is currently set to 0.\n"\
-			"If the Normal map has banding artifacts (extra stripes) set it to 1."
-		#define iUILogarithmic 0	
-	#endif
-
-	uniform int Depth_help <
-		ui_type = "radio"; ui_label = " ";
-		ui_text =
-			"\nThe right settings need to be set in the dialog that opens after clicking the \"Edit global preprocessor definitions\" button above.\n"
-		    "\n"
-			UPSIDE_DOWN_HELP_TEXT "\n"
-		    "\n"
-			REVERSED_HELP_TEXT "\n"
-		    "\n"
-			LOGARITHMIC_HELP_TEXT;
-	>;
-#else // "ui_text" was introduced in ReShade 4.5, so cannot show instructions in older versions
-	uniform bool bUIUseLivePreview <
-		ui_label = "Show live preview";
-		ui_tooltip = "Enable this to show use the preview settings below rather than the saved preprocessor settings.";
-	> = true;
-	
-	uniform int iUIUpsideDown <
-		ui_type = "combo";
-		ui_label = "Upside Down (Preview)";
-		ui_items = "RESHADE_DEPTH_INPUT_IS_UPSIDE_DOWN=0\0"
-		           "RESHADE_DEPTH_INPUT_IS_UPSIDE_DOWN=1\0";
-	> = RESHADE_DEPTH_INPUT_IS_UPSIDE_DOWN;
-	
-	uniform int iUIReversed <
-		ui_type = "combo";
-		ui_label = "Reversed (Preview)";
-		ui_items = "RESHADE_DEPTH_INPUT_IS_REVERSED=0\0"
-		           "RESHADE_DEPTH_INPUT_IS_REVERSED=1\0";
-	> = RESHADE_DEPTH_INPUT_IS_REVERSED;
-	
-	uniform int iUILogarithmic <
-		ui_type = "combo";
-		ui_label = "Logarithmic (Preview)";
-		ui_items = "RESHADE_DEPTH_INPUT_IS_LOGARITHMIC=0\0"
-		           "RESHADE_DEPTH_INPUT_IS_LOGARITHMIC=1\0";
-		ui_tooltip = "Change this setting if the displayed surface normals have stripes in them.";
-	> = RESHADE_DEPTH_INPUT_IS_LOGARITHMIC;
+uniform bool bUIShowOffset <
+    ui_label = "Blend Depth map into the image (to help with finding the right offset)";
+    ui_label_ja_jp = "透かし比較";
+    ui_tooltip_ja_jp = "補正作業を支援するために、画面効果を半透過で適用します。";
+> = false;
+
+uniform bool bUIUseLivePreview <
+    ui_category = "Preview settings";
+    ui_category_ja_jp = "基本的な補正";
+#if !ADDON_ADJUST_DEPTH
```

Comment:
Adding the following here would ensure the preview settings aren't shown by default on older ReShade versions too (like it was before):
```c++
#if __RESHADE__ < 50902
    ui_category_closed = true;
#elif !ADDON_ADJUST_DEPTH
    ui_category_toggle = true;
#endif
```